bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/NavigationImpl.java

```diff
@@ -94,7 +99,17 @@ private void initModel() {
             structureDepth = -1;
         }
         navigationRootPage = properties.get(PN_NAVIGATION_ROOT, currentStyle.get(PN_NAVIGATION_ROOT, String.class));
-        skipNavigationRoot = properties.get(PN_SKIP_NAVIGATION_ROOT, currentStyle.get(PN_SKIP_NAVIGATION_ROOT, true));
+        if (currentStyle.containsKey("rootLevel") || properties.containsKey("rootLevel")) {
```

Comment:
I'd use the constant instead of the `rootLevel` string.

Reply:
Done in https://github.com/adobe/aem-core-wcm-components/pull/836/commits/a051a2f1675345cb3ac0ae1ae4d6656170c1d947